alpakaConfig.cmake

```diff
@@ -282,7 +282,6 @@ IF(${ALPAKA_DEBUG} GREATER 1)
     MESSAGE(STATUS "Boost_MINOR_VERSION: ${Boost_MINOR_VERSION}")
     MESSAGE(STATUS "Boost_SUBMINOR_VERSION: ${Boost_SUBMINOR_VERSION}")
     MESSAGE(STATUS "Boost_LIB_DIAGNOSTIC_DEFINITIONS: ${Boost_LIB_DIAGNOSTIC_DEFINITIONS}")
-    MESSAGE(STATUS "Boost_LIBRARIES: ${Boost_LIBRARIES}")
```

Comment:
why did you remove that status? Is it truly empty now or was it already before?

Reply:
It is a duplicate. It is already printed in another line.